### ISSUE TITLE
Getting inconsistent data Excel wrapper  

### ISSUE THREAD

**Issue by Larry1957** [CoolProp/CoolProp#1539]
I am novice to CoolProp (v 6_1). I successfully installed the Windows version, with the aim to use CoolPro with the Excel high-level wrapper; and the test spreadsheet worked fine.  I then tried to compare the CoolProp result against the NIST REFPROP for the R134a. I downloaded a table of fluid properties form the NIST site and imported under the worksheet <NIST>. Then, using the same (P,T) point I used CoolProp (PropsSI) to check the same values for some fluid properties, storing the results in <CoolProp> worksheet.  Basically the results match, but every now and then  some values appear to be erroneous and discontinuous. I marked them in the attached spreadsheet.   I am doing something conceptulaly wrong
(NIST results are separately shown for liquid and vapor phases) when using PropsSI to recover fluid properties as function of just T and p?  Or are the internal table somehow corrupted.  I lack debugging skills and I would like avoiding to escalate up to Phyton debugging.  
Thanks for your support.
[Compare_R134a.xlsx](https://github.com/CoolProp/CoolProp/files/1246123/Compare_R134a.xlsx)

**Comment by jowr** [comment]
Low density values and high energy values indicate that you are getting a solution in the vapour phase. How close are you to the two-phase area?

**Comment by ibell** [comment]
The consistency plots for R134a look fine:
http://www.coolprop.org/_downloads/R134a1.pdf .  Can you try to use the
nightly build for Excel?  Is your Excel 32-bit or 64-bit?

**Comment by ibell** [comment]
Also, V is for viscosity, not volume: http://www.coolprop.org/coolprop/HighLevelAPI.html#table-of-string-inputs-to-propssi-function

**Comment by Larry1957** [comment]
Thanks to all.   I made a conceptual mistake trying to get the fluid parameter (i.e. density) from the pair (T,p) without specifying the phase state of the fluid, obtaining therefore a mix of results valid for liquid and vapor state. I checked with PhaseSI that the obtained results is in line with NIST estimation.
I attach a corrected version of the spreadsheet in which the density is separately estimated for liquid (Q=0) and for vapor (Q=1). Issue is closed
Thank
[Compare_R134a_02(24-Aug-2017).xlsx](https://github.com/CoolProp/CoolProp/files/1248087/Compare_R134a_02.24-Aug-2017.xlsx)
s again to all.
